docs/GetStarted.md

```diff
@@ -2,27 +2,35 @@
 ===
 
 ## **Installation**
+
 * __Dependencies__
 
-      python >= 3.5
-      git
-      wget
+  ```bash
+  python >= 3.5
+  git
+  wget
+  ```
+
+  python pip should also be correctly installed. You could use "python3 -m pip -v" to check in Linux.
 
-    python pip should also be correctly installed. You could use "python3 -m pip -v" to check in Linux.
-    
-    * Note: we don't support virtual environment in current releases.
+  * Note: we don't support virtual environment in current releases.
```

Comment:
It seems that virtual environment like conda is supported if we install it in the right way.

Reply:
There is no change on original content, I just changed indent of those lines. We can review content again, and get it correctly.